src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -49,6 +49,11 @@ enums:
         meaning: rdfs:Datatype
       rdf property:
         meaning: rdf:Property
+      sssom literal:
```

Comment:
The addition of `sssom literal` might require an explanation of what `rdfs literal` is intended for, and how those two differ.

Reply:
You are totally right.. I am wondering the same.. I dont think there is a point in rdfs literal, that was just some mistake I made in the early days of the schema. Should we:

1. Have "rdfs literal" do the job we assigned to "sssom literal" or
2. Remove "rdfs literal" to avoid confusion?

I am on the fence and cant decide. "sssom literal" sort of implies that it has a  "special place" in SSSOM. "rdfs literal is a bit ideosyncractic to the SemWeb world. But it is known, and "rdfs literal" is, in fact, expressing the same what "sssom literal is...

Reply:
I am mildly inclined to choose (1), i.e., forget about `sssom literal` and use the already existing `rdfs literal`. But I’d be fine with (2) as well.

Reply:
@graybeal thanks for weighing in, this pushed me firmly over the edge of indecision towards using rdfs literal!

Reply:
It opens another can of worms that the value, in a SSSOM TSV file, is going to be "rdfs literal" and in RDF, it is going to be `rdfs:Literal`. My bad for not defining `*_type` slots as linkml enums rather than entity references.. That seems to far gone now to change. At least, the model clearly binds the two together (using the `meaning` property).

Reply:
In any case, the issue here is fixed now, if you are satisfied @gouttegd please resolve conversation.

Reply:
> It opens another can of worms that the value, in a SSSOM TSV file, is going to be "rdfs literal" and in RDF, it is going to be rdfs:Literal.

I don’t see how that is a problem. The same value will be serialised differently in different serialisation formats. So what?

Reply:
No need to restart that debate now.. I let a formalism (LinkML) determine the serialisation, not right. There are good reasons to use rdfs:Literal instead of "rdfs literal", like the fact that it is instantly recognisable as to what it means.

But that ship has sailed, and yes, there are good reasons to use "human readable" enums here in the *_type fields as well! So, its not too bad.

Reply:
I am not debating anything, I just don’t see what makes you think that this is a “can of worms” that the same value can have different serialisations. For me this is a non-issue.

Reply:
Yes, didnt mean to say, "no need for you Damien to debate with me", I meant to say "no need to resurface this past decision and debate it with my younger self :P Non-issue indeed!